resdk/tests/functional/base.py

```diff
@@ -79,6 +79,15 @@ def get_gtf(self, collection=None, species='Homo sapiens',
             **collections
         )
 
+    def get_samplesheet(self):
```

Comment:
Move this to the actual test case as this is not a general file, but is useful only for specific example.